src/rg3d/ui/border.md

```diff
@@ -1 +1,27 @@
-# Border (WIP)
\ No newline at end of file
+# Border
+
+To create a simple thick border you should do something like this:
+
+```rust
+# extern crate rg3d;
+# use rg3d::{
+#     core::pool::Handle,
+#     gui::{button::ButtonBuilder, widget::WidgetBuilder, border::BorderBuilder, UiNode, UserInterface},
+# };
+fn create_border_with_button(ui: &mut UserInterface) -> Handle<UiNode> {
+    BorderBuilder::new(WidgetBuilder::new()
+            .with_child(
+                ButtonBuilder::new(WidgetBuilder::new())
+                .with_text("Click me")
+                .build(&mut ui.build_ctx()),
+            )
+    )
+    .with_stroke_thickness(left: 1.0, right: 1.0, top: 1.0, bottom: 1.0)
```

Comment:
This won't compile, you've missed `Thickness { .. }`

Reply:
Still needs to be fixed, replace `left: 1.0, right: 1.0, top: 1.0, bottom: 1.0` with `Thickness { left: 1.0, right: 1.0, top: 1.0, bottom: 1.0 }`